tests/rptest/tests/log_compaction_test.py

```diff
@@ -256,7 +256,6 @@ def compaction_has_completed():
 
         # There should be no dirty segments left
         assert self.get_dirty_segment_bytes() == 0
-        assert self.get_closed_segment_bytes() > 0
```

Comment:
so i'm curious why not disable this assertion for the topic,delete configuration? it's passed into the test context from the @.matrix parameterization so the information is available.